course/layouts/lists/single.html

```diff
@@ -2,13 +2,17 @@
 
   {{- $bundle := . -}}
   {{- $uids := .Params.resources.content -}}
+  {{- $resources := slice -}}
 
   {{ partial "course_content.html" . }}
   <div class="mb-5">
     <div class="mb-3 collection-description">
       {{ $bundle.Description | .RenderString }}
     </div>
-      {{ $resources :=  where .Site.Pages "Params.uid" "in" $uids }}
-      {{- partial "resource_list.html" (dict "resources" $resources) -}}
+      {{ range $id := $uids }}
+        {{ $page := where $.Site.Pages "Params.uid" $id }}
+        {{$resources = $resources |  append (index $page 0) }}
+      {{ end }}
+      {{- partial "resource_list.html" (dict "resources" $resources "sort" false) -}}
```

Comment:
Here and a couple of other places, I see you're using the `{{-` syntax with the hypens on the opening and closing tags.  I'm certainly guilty of misusing these myself, but I think moving forward we should pay attention to where and how these are used and clean up where they aren't supposed to be used.  Basically, what that does is [trim whitespace](https://gohugo.io/templates/introduction/#whitespace), not only in the HTML being generated by whatever's inside the brackets, but directly adjacent to it as well.  If you look at the examples in the Hugo docs you should see what I mean.  Using the trim delimiters excessively doesn't really hurt anything,as all it does is remove whitespace, but it makes the resulting HTML a bit more messy than it could have been.  Basically, as a rule of thumb we should only be using the trim delimiters in situations where it is needed to trim whitespace.